### PR TITLE
Add info about Device Farm in the table

### PR DESCRIPTION
**Pull request by artemnikitin**
I added information about Device Farm to the table with info about similar services from other providers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,6 +264,8 @@ Many services within AWS can at least be compared with Google Cloud offerings or
 | Git hosting                   | CodeCommit                                                                   |                              |                 |            | GitHub, BitBucket                 | GitLab                                                     |
 | User authentication           | Cognito                                                                      |                              |                 |            |                                   | oauth.io                                                   |
 | Mobile app analytics          | Mobile Analytics                                                             |                              |                 |            | Mixpanel                          |                                                            |
+| Mobile apps testing        | Device Farm                                                           |Cloud Test Lab                              |                 |Xamarin Test Cloud            | BrowserStack, Sauce Labs, Testdroid                          |                                                            |
```
Comment (ThanosBaskous) [inline README.md]: Consistency with previous line: "Mobile app testing"

**Comment by artemnikitin** [comment]
Fixed

**Comment by ThanosBaskous** [comment]
Thanks @artemnikitin!

**Comment by jlevy** [comment]
Awesome, thanks @artemnikitin ! Btw do you have a lot of experience with Device Farm? More info/tips on it would be great — if there are enough to start a section, we can do that.

**Comment by artemnikitin** [comment]
I plan to add more info about Device Farm on this week.

**Pull request merged**